index.html

```diff
@@ -1,19 +1,275 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
     <title>Karma</title>
-    <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
-    <link rel="stylesheet" href="css/normalize.css">
-    <link rel="stylesheet" href="css/style.css">
-    <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
-</head>
-<body>
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto:400,500,300"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link rel="stylesheet" href="css/normalize.css" />
+    <link rel="stylesheet" href="css/style.css" />
+    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
+  </head>
+  <body>
+    <header class="header">
+      <img src="img/karma-logo.svg" alt="Karma logo" class="header__logo" />
+      <div>
+        <nav>
+          <ul class="navigation__list">
+            <li>
+              <a class="navigation__link__karma" href="#">Meet Karma</a>
+            </li>
+            <li>
+              <a href="#">How It Works</a>
+            </li>
+            <li>
+              <a href="#">Store</a>
+            </li>
+            <li>
+              <a href="#">Blog</a>
+            </li>
+            <li>
+              <a href="#">Help</a>
+            </li>
+            <li>
+              <a href="#">Login</a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+
+    <div class="content">
+      <main class="landing_page">
+        <div class="introduction">
+          <h1>Introducing Karma</h1>
+          <p>Bring WiFi with you, everywhere you go.</p>
+          <button>Learn More</button>
+        </div>
+      </main>
+    </div>
+
+    <h2>Everyone needs a little Karma.</h2>
+
+    <section class="icons__section">
+      <div class="icon">
+        <img
+          src="img/icon-devices.svg"
+          alt="three devices"
+          class="icon__logo"
+        />
+        <p>Internet for all devices</p>
+      </div>
+      <div class="icon">
+        <img src="img/icon-coffee.svg" alt="coffee cup" class="icon__logo" />
+        <p>Boost your productivity</p>
+      </div>
+      <div class="icon">
+        <img src="img/icon-refill.svg" alt="charger icon" class="icon__logo" />
+        <p>Pay as You Go</p>
+      </div>
+    </section>
+    
+    <section class="features">
+      <div class="rectangle">
+        <p class="quote">
+          Wherever I am, I just don't worry about my connection anymore!
+        </p>
+        <button href="#" class="button_color">Get Karma today</button>
```

Comment:
Do you want to try linking this button to the store page?

Reply:
I had it done in VSC and don't know why didn't add it here, I corrected it now.